### ISSUE TITLE
Stock for admin shopkeeper isn't kept

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#63]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/53

**Originally posted by BEQOsNtDi6xRDGzqIAT7 (Nov 17, 2012):** 

What steps will reproduce the problem?
1. Create admin shopkeeper (look at space and type /shopkeeper)
2. Load up shopkeeper with stuff
3. Close shopkeeper inventory
4. Try to tradeWhat is the expected output? What do you see instead?
Expected: Trades show up and ready to shop
Actual: Trades are empty and all items are moved back to your inventory when you recheck via shift-right-clickWhat version of the product are you using?
1.9 test dev for 1.4.4Do you have an error log of what happened?
No errors are generated for this event.Please provide any additional information below.
None available.

**Comment by blablubbabcDEV** [comment]
**Originally commented by nisovin (Nov 17, 2012):** 

It works fine for me. Are you sure you're setting up the trades correctly? Did you follow the instructions provided on the main page?

**Comment by blablubbabcDEV** [comment]
**Originally commented by BEQOsNtDi6xRDGzqIAT7 (Nov 17, 2012):** 

@nisovin:
I did it multiple times before in the same way successfully with the previous version. Why would i change what worked?
Also, only newly created shopkeepers are affected by this. Old ones continue to work.
Edit: ...hold the phone. Ok, the thing i said previously? Discard that. Turns out that this is indeed a user error. Ticket can be closed.
(Specifically, putting things into the second row, not the third. So much for having done it multiple times before... tch.)

            Edited Nov 17, 2012
